.github/ISSUE_TEMPLATE/05-dev-days-project.yml

```diff
@@ -0,0 +1,74 @@
+name: Indicate your project is participating in Dev Days
+description: | 
+  For hosted projects only, use this form to indicate that your project will be participating in Dev Days.
+title: "Dev Days 2024 Project Application - PROJECTNAME"
+labels: ["help wanted"]
+assignees: 
+  - yarille
+  - jmertic
+  - carolalynn
+  - lgritz
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ![](https://artwork.aswf.io/other/dev-days/2024/color/dev-days-2024-color.svg)
+
+        Thank you for your interest in being a project wanting to participate in Dev Days 2024!
+        For hosted projects only, use this form to indicate that your project will be participating in Dev Days.
+
+        Please update `PROJECTNAME` in the title with your project name.
+  - type: input
+    attributes:
+      label: URL to your Dev Days 2024 landing page
+      description: |
+        This URL can be a webpage, wiki page, or file in your code repository, that outlines...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: URL of issues for participants to work on
+      description: |
+        Projects should label issues with 'good first issue' and 'help wanted', and ensure 
+        that they are unassigned and no more than one year old. To validate, go to 
+        [CLOTributor](https://clotributor.dev/search?foundation=aswf&good_first_issue=true).
+
+        These issues should align with the Academy Software Foundation TAC's
+        [best practices for good first issues]()
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Communication channel(s) for partipants
+      description: |
+        What are the best communications channel(s) for Dev Days particpants to use? This is
+        likely your primary Slack channel or mailing list, but you could also setup a seperate
+        communication channel if that would be preferred.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Project meeting(s) for Dev Days partipants to join
+      description: |
+        What project meeting(s) should a potential Dev Days participant join if they want to 
```

Comment:
Wording suggestion:

"Please include at least one scheduled meeting Dev Days participants for your project should attend. This could be a regular TSC meeting, or a separately scheduled call."

Inputs: Date/Time, Zoom link